test/testrunner.py

```diff
@@ -430,6 +430,12 @@ def main():
             report = os.path.join(os.getcwd(), "test", f"{test_type}.xml")
             sm_utils.generate_empty_report(report, test_type, "neuron")
             return
+        if "huggingface" in dlc_images:
```

Comment:
this will skip for all HF - it looks like HF pytorch has local mode tests, same with HF tf inference. We only want to skip HF tf training. "huggingface-tensorflow-training" is likely what you want to skip.

Also please remove the "placeholder" test as well as the whole "local" directory under huggingface_tensorflow, since it is not providing any value and just adding confusion.